asteroid/models/demask.py

```diff
@@ -178,3 +160,31 @@ def get_model_args(self):
         }
         model_args.update(self.fb_kwargs)
         return model_args
+
+
+def _build_masker_nn(
```

Comment:
I wonder if this is the good way to go. 
I know the architecture is really simple, but should this be a hidden function? 
For other models, we have a `nn.Module` that can be used with other features than asteroid filterbanks. 
This might be related to @popcornell 's comments on asteroid's source code organization.

Reply:
maybe having it as a method would be better. 

> we have a nn.Module that can be used with other features than asteroid filterbanks.
I must have missed this. What is this module ?

Reply:
> maybe having it as a method would be better.

A method in the class? Or a function outside (like now)? public or private? 

> I must have missed this. What is this module ?

Sorry, it was not clear. I mean the masker architectures e.g `TDConvNet` can be used on anything and is exposed to the user. Here, the masker architecture from `DeMask` is not.

Reply:
> A method in the class? Or a function outside (like now)? public or private?

private Method in the class. 

> Sorry, it was not clear. I mean the masker architectures e.g TDConvNet can be used on anything and is exposed to the user. Here, the masker architecture from DeMask is not.

OK, well it is a simple MLP the masker architecture.... i don't know how much is useful for other tasks. 

BTW, can't in principle `BaseENcoderMaskerDecoder` be used ? Or it can't because of difference for STFT mag vs free for example.

Reply:
This PR makes that we use `BaseEncoderMaskerDecoder` for `DeMask` now. 

> OK, well it is a simple MLP the masker architecture.... i don't know how much is useful for other tasks.

I know that it's a very simple architecture but I wonder if it makes a difference. 

Too many choices for today for me :joy: